scio-core/src/main/scala/com/spotify/scio/coders/DerivedCoder.scala

```diff
@@ -77,23 +77,23 @@ private object Derived extends Serializable {
 }
 
 trait LowPriorityCoderDerivation {
-  import magnolia._
+  import magnolia1._
 
   type Typeclass[T] = Coder[T]
 
-  def combine[T](ctx: CaseClass[Coder, T]): Coder[T] =
```

Comment:
What's the logic on this rename?

Reply:
`magnolia1` changed its API:
- `combine` -> `join`
- `dispatch` -> `split`

The magnolia macro expects those functions to be defined. This code shouldn't be used directly by users anyway